content/docs/composition-vs-inheritance.md

```diff
@@ -164,9 +166,9 @@ class SignUpDialog extends React.Component {
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZbYa?editors=0010)
 
 ## So What About Inheritance? {#so-what-about-inheritance}
+'Facebook' में हम 'React' का हज़ार जगह प्रयोग करते है, परन्तु हमे कोई भी ऐसा केस नहीं मिला जहाँ हम आपको कंपोनेंट्स inheritance hierarchies बनाने की सलाह दे।
 
-At Facebook, we use React in thousands of components, and we haven't found any use cases where we would recommend creating component inheritance hierarchies.
-
-Props and composition give you all the flexibility you need to customize a component's look and behavior in an explicit and safe way. Remember that components may accept arbitrary props, including primitive values, React elements, or functions.
+'props' एवं 'composition' आपको हर प्रकार का लचीलापन देता है ताकि आप अपने कौम्पोनॅन्ट का देख दिखाव एवं बिहेवियर बदल सके एक सुरक्षित एवं बिना किसि उलझन के।
+याद रखिये की एक कौम्पोनॅन्ट 'Arbitiary props', जिसमे 'primitive' वैल्यू हो, 'react' एलिमेंट्स या फिर फंक्शन्स को स्वीकार कर सकता है।
 
-If you want to reuse non-UI functionality between components, we suggest extracting it into a separate JavaScript module. The components may import it and use that function, object, or a class, without extending it.
+अगर आप अपने कॉम्पोनेन्ट के बीच कोई भी 'non-UI' फंक्शनलिटी का प्रयोग करना चाहते है, हम कहेंगे की आप एक अलग जावास्क्रिप्ट मॉडल्स का प्रयोग करें क्योंकि कोई भी कॉम्पोनेन्ट इसको इम्पोर्ट कर सकता है एवं इसको 'function' ,'object', या फिर 'class' की तरह प्रयोग कर सकता है बिना इसको एक्सटेंड करे।
```

Comment:
`class` => `क्लास`

Reply:
@arshadkazmi42  On glossary page it was written, No need to translate the class word. Should I do it?

Reply:
Oh, I completely forgot about it. You can leave it as it,
just remove `'` single quotes from words even if you are not translating it

Reply:
You have translated `class` here as per our discussion we are going to keep it as it is

Reply:
This is not done at this line
https://github.com/reactjs/hi.reactjs.org/pull/28/files#diff-8797b484bef460165b0ef9c6e8f90ec6R172

We are not translating class
change `क्लास` => `class`